addusers.py

```diff
@@ -401,19 +403,12 @@ def parse_rows(rows, select_user=None):
                 # this could happen if we delete a project or change its name
                 # but forget to update the form
                 raise ItemNotFoundError('Project', content[project].name)
-        except ItemExistsError:
-            print ("skipping existing project marked "
-                   "as new: {}").format(content[project].name)
-            continue
-            # FIXME: Actually handle this, it should just skip this project
-            # and do error reporting.  Need to consider whether to create
-            # the associated users or not.
+        except (ItemExistsError, ItemNotFoundError) as e:
+            # FIXME:  Need to some way to handle users associated with the
+            # skipped project
```

Comment:
Yes, we still print this information, but later.  For the change we add the rows and the error message to the list bad_rows (line 411 below).  Then in line 478 a summary of all the bad rows is printed.

This way of error reporting is kind of clunky, and will likely be changed later, but for now this standardizes the error messages which will help the UMass Helpdesk staff interpret the output.  Other errors were already handled this way, these 2 were temporary print statements I added in place of real error handling.